src/scene/renderer/forward-renderer.js

```diff
@@ -229,6 +233,14 @@ class ForwardRenderer extends Renderer {
                 this.shadowCascadeDistancesId[cnt].setValue(directional._shadowCascadeDistances);
                 this.shadowCascadeCountId[cnt].setValue(directional.numCascades);
                 this.lightShadowIntensity[cnt].setValue(directional.shadowIntensity);
+                this.lightSizeId[cnt].setValue(directional.lightSize);
+
+                const cameraParams = new Float32Array(4);
```

Comment:
I'd like to avoid the allocation in all 3 cases, and store the array / float32array on the light itself, similarly to how _shadowRenderParams is handled.

Reply:
Done